### PR TITLE
Improve scene command and do some cleanup

### PR DESCRIPTION
**Pull request by tamilpp25**
Supports both floorId and planeId as same argument now

### CODE REVIEW

File: src/commands/scene.ts [hunk 1]
```diff
@@ -13,25 +13,49 @@ export default async function handle(command: Command, session: Session) {
...
-    const planeID = MazePlaneExcel.fromPlaneId(parseInt(command.args[0]))
+    if(command.args.length == 0){
+        c.log("Usage: /scene <planeID|floodID>");
+        return;
+    }
+
+    let floorId = 10001001 // Default floor data
+
+    if(command.args[0].length === 5){//PLANE ID LOGIC
+        const planeID = MazePlaneExcel.fromPlaneId(parseInt(command.args[0]))
```
Comment (memetrollsXD) [inline src/commands/scene.ts:24]: Maybe you can simplify this code to:

Excel.fromId(xx) || Excel.fromPlaneId(xxx)

**Review by memetrollsXD — APPROVED** [review]

File: src/commands/scene.ts (hunk 1 above)
Comment (tamilpp25) [reply src/commands/scene.ts:24]: I'd have to make a function for it then since .fromId doesn't exist , should I?

Comment (memetrollsXD) [reply src/commands/scene.ts:24]: Yeah sure if that seems better for you, just a small suggestion

**Comment by tamilpp25** [comment]
code broke somewhere I will remake pr

**Pull request closed without merging**